pkg/kubelet/rkt/pod_info.go

```diff
@@ -38,6 +38,8 @@ const (
 	Deleting       = "deleting" // This covers pod.isExitedDeleting and pod.isDeleting.
 	Exited         = "exited"   // This covers pod.isExited and pod.isExitedGarbage.
 	Garbage        = "garbage"
+
+	exitCodePrefix = "app-"
```

Comment:
The output for `rkt status $UUID` is now:
```
state=exited
pid=-1
exited=true
app-etcd=0
app-redis=0
```

So `app-` is the prefix before the app name for each app's exit code. (We definitely need some structured data here).

I think I need to add some doc here.